src/Test/Utilities/Portable/Diagnostics/DescriptorFactory.cs

```diff
@@ -14,15 +14,16 @@ public static class DescriptorFactory
         /// </summary>
         /// <remarks>
         /// Returned <see cref="DiagnosticDescriptor"/> has
-        /// - empty <see cref="DiagnosticDescriptor.Title"/>, <see cref="DiagnosticDescriptor.MessageFormat"/> and <see cref="DiagnosticDescriptor.Category"/>
+        /// - empty <see cref="DiagnosticDescriptor.Title"/> and <see cref="DiagnosticDescriptor.Category"/>
+        /// - <see cref="DiagnosticDescriptor.MessageFormat"/> set to <paramref name="id"/>
         /// - <see cref="DiagnosticDescriptor.DefaultSeverity"/> set to <see cref="DiagnosticSeverity.Hidden"/>
         /// - <see cref="WellKnownDiagnosticTags.NotConfigurable"/> custom tag added in <see cref="DiagnosticDescriptor.CustomTags"/>.
         /// </remarks>
         /// <param name="id">The value for <see cref="DiagnosticDescriptor.Id"/>.</param>
         /// <returns>A <see cref="DiagnosticDescriptor"/> with specified <see cref="DiagnosticDescriptor.Id"/>.</returns>
         public static DiagnosticDescriptor CreateSimpleDescriptor(string id)
         {
-            return new DiagnosticDescriptor(id, title: "", messageFormat: "", category: "",
+            return new DiagnosticDescriptor(id, title: "", messageFormat: id, category: "",
```

Comment:
is this appropriate?  seems... strange...

Reply:
💭 It's easier than further work to specialize the behavior for a test-only analyzer.